### PR TITLE
IAP-5278: fix customize 404 page section

### PR DESCRIPTION
**Pull request by kvedenskii-axway**
https://deploy-preview-2376--axway-open-docs.netlify.app/docs/apim_administration/apiportal_admin/customize_getting_started/#customize-404-error-page

### CODE REVIEW

**Review by dginevaAxway — APPROVED** [review]

File: content/en/docs/apim_administration/apiportal_admin/customize_getting_started.md [hunk 1]
```diff
@@ -222,36 +222,36 @@ You can customize API Portal standard footer `Copyright © YYYY Axway` to displa
 5. Use the text editor to enter the text.
 6. Click **Save**.
 
-## Customize 404 error page
+## Customize the 404 error page
```
Comment (andreamussap) [inline content/en/docs/apim_administration/apiportal_admin/customize_getting_started.md:225]: I was re-reading this and I realized that having "Create styles" under "Customize 404 page" Doesn't seem right. These things don't seem to be related. So, I think we should change this structure. Can I go ahead with this change?

Comment (dginevaAxway) [reply content/en/docs/apim_administration/apiportal_admin/customize_getting_started.md:225]: @andreamussap I think that duplicate , create style for home page and all other pages are the steps to apply customization for error page. I mean, first you need to duplicate the style and then you need to modify it in the way you want for home and all other pages. Without this steps error page is not appearing in the way the customer wants in all places, but will appear by it's default look and feel. In short they all are steps in the customize error page section. @kvedenskii-axway

**Review by dginevaAxway — APPROVED** [review]
